src/commands/cluster-replicate.json

```
@@ -1,21 +1,50 @@
 {
     "REPLICATE": {
-        "summary": "Configure a node as replica of a primary node.",
+        "summary": "Configure a node as replica of a primary node or detaches replica from primary.",
         "complexity": "O(1)",
         "group": "cluster",
         "since": "3.0.0",
-        "arity": 3,
+        "arity": -3,
         "container": "CLUSTER",
         "function": "clusterCommand",
+        "history": [
+            [
+                "9.0",
```

These files seem to use x.y.z format in all version numbers.

```suggestion
                "9.0.0",
```

---

Done